nucypher/cli/actions.py

```diff
@@ -138,11 +140,15 @@ def load_seednodes(emitter,
             teacher_uris.extend(seednode_uris)
 
```

Comment:
Instead of the log message at L118, what do you think of adding the following here at L141
```suggestion

            emitter.message(f'Connecting to preferred seednodes for {domain}: {teacher_uris}', color='yellow')
```